test/e2e-tests.js

```diff
@@ -539,6 +539,10 @@ lab.experiment("core rendering-info", () => {
       expect(response.result.scripts[0].content).to.be.equal(
         'console.log("functionGeneratedScriptContent");'
       );
+
+      expect(response.result.sophieModules[0].name).to.be.equals(
```

Comment:
The test cases in this group are here to test the feature additionalRenderingInfo as functions. For example: https://github.com/nzzdev/Q-server/blob/dev/test/config/tools.js#L25-L29
You seem to mix this here with something else.

Can you create a new test group called "correctly merges additionalRenderingInfo from tool endpoint and target config" and add some test cases in there?

That would make it more clear what the tests actually test.